### ISSUE TITLE
Cache : enable the user to specify if caching should be done for table or not

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#2664]
Some internal tables may hold data that should not be attempted to be cached:
- auditing (when sstables are used as the backend) 

We have also seen some cases in which customers have two tables and one should be cached while the other shouldn't.

The current approach of caching everything is simple - yet enabling the user (or system) to specify caching hints can help.

The default should be that cache is enabled.

**Comment by tzach** [comment]
Similar to C* 2.1 cache properties
```
keys: ALL or NONE
rows_per_partition: number of CQL rows (N), NONE, or ALL
```
http://docs.datastax.com/en/cql/3.1/cql/cql_reference/tabProp.html#tabProp__moreCaching

**Comment by julienlau** [comment]
+1

**Comment by slivne** [comment]
@avikivity comments ?

**Comment by tzach** [comment]
@slivne duplicate of #755 ?

**Comment by slivne** [comment]
this was implemented in 2d727114ed0a029a84828151c7f3acdc7e09fb3f

by default cache is enabled

users can disable the cache using ```WITH CACHE={'enabled': false}```

**Comment by slivne** [comment]
since 4.2.0
